workitem/link/link_repository.go

```diff
@@ -97,6 +97,31 @@ func (r *GormWorkItemLinkRepository) ValidateCorrectSourceAndTargetType(ctx cont
 	return nil
 }
 
+// CheckParentExist returns error if there is an attempt to create more than 1 parent of a workitem.
+func (r *GormWorkItemLinkRepository) CheckParentExist(ctx context.Context, sourceID, targetID uint64, linkTypeID uuid.UUID) error {
```

Comment:
Ah, :) that's what you mean. I agree with @baijum then. Thanks for explaining what you mean. I didn't scroll to the horizontal end of the function to notice that it actually only returns error ;)